opendevin/action/browse.py

```diff
@@ -24,29 +36,19 @@ async def run(self, controller: 'AgentController') -> BrowserOutputObservation:
         if not asked_url.startswith('http'):
             asked_url = os.path.abspath(os.curdir) + self.url
         try:
-            async with async_playwright() as p:
-                browser = await p.chromium.launch()
-                page = await browser.new_page()
-                response = await page.goto(asked_url)
-                try:
-                    # domcontentloaded: Wait for the DOMContentLoaded event to be fired.
-                    # load: Wait for the load event to be fired.
-                    # networkidle: Wait until there are no more network connections
-                    await page.wait_for_load_state('networkidle', timeout=3000)
-                except TimeoutError:
-                    pass
-                # content = await page.content()
-                inner_text = await page.evaluate('() => document.body.innerText')
-                screenshot_bytes = await page.screenshot(full_page=True)
-                await browser.close()
-
-                screenshot_base64 = base64.b64encode(screenshot_bytes).decode('utf-8')
+            action_str = f'goto("{asked_url}")'
```

Comment:
<s> Then, in the child class, you can just do something like `super().run(f'goto("{asked_url}")')`?
This will allow us to extend different BrowserAction more easily IMO, by simply subclassing and tweak the `action_str` according to `browsergym` documentation </s>

See below comment for Browser class.

Reply:
It will be nice if we can put a URL to the documentation of supported actions in the code